src/Stratis.Bitcoin.Features.SignalR/Broadcasters/WalletInfoBroadcaster.cs

```diff
@@ -84,7 +75,7 @@ protected override IEnumerable<IClientEvent> GetMessages()
                         CreationTime = wallet.CreationTime,
                         LastBlockSyncedHeight = wallet.AccountsRoot.Single().LastBlockSyncedHeight,
                         ConnectedNodes = this.connectionManager.ConnectedPeers.Count(),
-                        ChainTip = this.chainIndexer.Tip.Height,
+                        ChainTip =  this.consensusManager.HeaderTip,
```

Comment:
This is not only the chain tip, it's the tip of the best connected peer (unless no peers are connected)